### PR TITLE
Microverse project

### PR DESCRIPTION
**Pull request by Theophileaseh**
In this project, I created a repository, a branch, a .html file, a .css file, a test.md file, added the test.md file to .gitignore file and created a pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+  </head>
```
Comment (elmar8287) [inline index.html:4]: - [ ] Please, consider correct creation of HTML file structure. You missed some meta tags also. [Here](https://www.webfx.com/blog/web-design/html5-template/) you can find the information on how HTML file structure should be.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  <body>
+    <header>
+      <h1>Hello Microverse!</h1>
```
Comment (elmar8287) [inline index.html:7]: - [ ] Please, consider giving a class name for an H1 tag and style in the style.css file, instead of directly styling the HTML tag.

**Review by elmar8287 — COMMENTED** [review]
Hi @Theophileaseh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.
- [ ] Please, consider NOT MERGE the branch related to PR, which you are going to submit. Do it after you will get approved from review 👍 It will make more professional your submission process.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — COMMENTED** [review]
Hi @Theophileaseh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
- [optional] In the README file please change the project name.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**